### ISSUE TITLE
Can Fabric work with more than one positive or negative latent feedback?

### ISSUE THREAD

**Issue by EricRollei** [ssitu/ComfyUI_fabric#9]
ChrisGoringe's Image Picker / Chooser  and Use Everywhere (UE) node sets enabled an interesting workflow for use with Fabric.     We have been testing this today and I built a 4 round Fabric workflow with his multi-latent chooser node. that allows the workflow to pause after each step until the user selects the positive and negative images from a batch - then those positive and negative latents are sent to the next round as feedback to the Fabric sampler.   I got some strange results playing with this today where it was almost like going further away from the feedback - like exactly reversed.   But I wonder if it's not possible to send more than one latent to either pos or neg latent feedback?   
I'm attaching the workflow in a zip in case you want to check it out, but you'll need the UE and Image picker node sets from ChrisGoringe 
https://github.com/chrisgoringe/cg-image-picker
https://github.com/chrisgoringe/cg-use-everywhere

[Fabric-4-round-iterative-UEnodes.zip](https://github.com/ssitu/ComfyUI_fabric/files/12780083/Fabric-4-round-iterative-UEnodes.zip)

Also here are some screen grabs to show you it as well.  First is of the basic round where you see the image chooser and multi-latent nodes that allow the user to select the pos and neg, then hit "go" to advance to the next round. You can also see the pos and neg feedback choices.  The prompt was a "beautiful landscape of fall trees, a meadow with a stream running and a foot bridge that crosses it. "

![image](https://github.com/ssitu/ComfyUI_fabric/assets/97568200/8fbbc9ce-ac4b-4c48-afb0-40e4e5aacc08)

here's what was generated for Round 2 choices -  way crazy saturation and further away from the start

![image](https://github.com/ssitu/ComfyUI_fabric/assets/97568200/f5ce305a-1253-432d-bee3-f4587f03bcb3)

and round 3 even further.   It's almost like the positive and negative feedback was reversed.

![image](https://github.com/ssitu/ComfyUI_fabric/assets/97568200/847ce0fc-5e0d-44b6-ad92-79c68dd60f83)

But my guess is that this doesn't work with more than one latent feed back and I wondered if that was true and if not why not?  Could it be made to take more than one feedback image?

**Comment by chrisgoringe** [comment]
I think you can send multiple latents to the pos and neg. 

I just did a quick test, in which the seeds were all fixed throughout:
- generated six images in the first stage (identical throughout because seeds fixed)
- selected latent 1 as positives and 6 as negative, and passed it through the second stage. Call this image p1n6
- repeated to generate p2n6, p1n5, and p2n5. All four images were different, as expected.
- I then selected 1 & 2 as positive, and 5 & 6 as negative. Call this p12n56
- p12n56 was different from all the other images. So having multiple latents definitely had an effect 
- I also tried generating p1n5, and then using accumulate to add 2 and 6 - this gave an identical image to p12n56, so the accumulate works as expected.

I note that in the tips, it says "Pay attention to the pos/neg weights. The default value of 1.0 is probably too high in most cases." so I wonder if that's the issue?

**Comment by EricRollei** [comment]
Yes, possibly the weights are an issue since in that workflow, I had no easy way to set them.

**Comment by ssitu** [comment]
The negative weight is better set to a smaller value. I found that negative images are taken very literally. So if you put a decent looking image in the negatives, you will get a very bad result if your weight is 1.0.

**Comment by lzardy** [comment]
I use batch image -> vae encode -> pos/neg latent.
